src/test/java/com/wikia/webdriver/common/testnglisteners/BrowserAndTestEventListener.java

```diff
@@ -83,7 +83,7 @@ public void afterNavigateTo(String url, WebDriver driver) {
         // HACK FOR DISABLING NOTIFICATIONS
         try {
           new JavascriptActions(driver).execute("$('.wds-banner-notification__close').click()");
-          new JavascriptActions(driver).execute("$('.sprite.close-notification')[0].click()");
+          new JavascriptActions(driver).execute("$('#WikiaNotifications .sprite.close-notification').click()");
         } catch (WebDriverException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1166)